set.mm

```diff
@@ -4300,10 +4300,25 @@ inference eliminates the need for a syllogism ( ~ syl ) -type inference
       ( ex com12 ) ABCABCDEF $.
   $}
 
+  ${
+    expdcom.1 $e |- ( ph -> ( ( ps /\ ch ) -> th ) ) $.
+    $( Commuted form of ~ expd .  (Contributed by Alan Sare, 18-Mar-2012.)
+       Shorten ~ expd .  (Revised by Wolf Lammen, 28-Jul-2022.) $)
```

Comment:
That line seems redundant since the mention of your shortening already appears in the comment of ~expd.

Reply:
I added it to indicate the date and incentive of the latest proof change.  I am not in particular keen on letting my name appear there.  Something like 'Last modified on 28-Jul-2022.' would just be fine, and help anyone interested in the evolution of a theorem proof.  But that has to be agreed upon.

Reply:
Ah, it was for the date, I see. Do as you see fit, and you can merge.